GFS_layer/GFS_typedefs.F90

```diff
@@ -1027,6 +1027,54 @@ module GFS_typedefs
     real (kind=kind_phys), pointer :: phy_f2d  (:,:)   => null()  !< 2d arrays saved for restart
     real (kind=kind_phys), pointer :: phy_f3d  (:,:,:) => null()  !< 3d arrays saved for restart
 
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
The `tbd_type`, meaning "To Be Determined" type, probably needs to be renamed now that it has an actual purpose. How about "statemid_type" to distinguish from either "statein_type" or "stateout_type"?